Makefile

```diff
@@ -55,7 +62,11 @@ boiler:
 
 cri-containerd: check-gopath
 	$(GO) build -o $(BUILD_DIR)/$@ \
-		$(PROJECT)/cmd/cri-containerd
+	-ldflags '-X $(PROJECT)/pkg/version.criContainerdVersion=$(VERSION)' \
+	$(PROJECT)/cmd/cri-containerd
+
+test: 
+	go test -timeout=1m -v -race ./pkg/... $(BUILD_TAGS)
```

Comment:
I believe go test also supports -ldflags so if you add it your unit test should be able to check ver on the tag..

Maybe make a variable for this since you would be using it twice.